orttraining/orttraining/python/training/ortmodule.py

```diff
@@ -169,6 +174,8 @@ def forward(ctx, *inputs, **kwargs):
                 def backward(ctx, *grad_outputs):
                     '''Performs backward pass based on grad wrt module output
                     '''
+                    if not ctx.run_info:
+                        raise RuntimeError('ctx.run_info not set in forward')
```

Comment:
```suggestion
                    assert ctx.run_info is not None, 'forward() or __call__() methods must be called before backward()'
```
We can convert it to an `assert` as there is no way `ctx.run_info` is not set if forward is called

Reply:
Using assert now